_content/en/03-dont/03-be-on-guard-against-false-hope.md

```diff
@@ -3,4 +3,4 @@
 **against false hope of "remedies"**
 
 Be good to your body. Expect companies to capitalize on your fear. Don't buy products that claim to protect or heal you at this point. Currently
-available products like ibuprofen and paracetamol can be useful to lessen some symptoms; however so far, nothing has been proven to do above random noise. Don't believe individuals who say that crystals, essential oils, etc. have helped them; the folks sharing these products may have the best intentions, or they may be predatory. But the idea is the same: many patients spontaneously do get better on their own because most cases are mild and resolve without pharmaceutical intervention. Instead of worrying about remedies, eat good food. Exercise. Meditate. Rest. Don't neglect your mental health and don't use your anxiety as an excuse for poor behavior towards others.
```

Comment:
More recent information: https://www.fda.gov/drugs/drug-safety-and-availability/fda-advises-patients-use-non-steroidal-anti-inflammatory-drugs-nsaids-covid-19?utm_campaign=FDA%20advises%20patients%20on%20use%20of%20NSAIDs%20for%20COVID-19&utm_medium=email&utm_source=Eloqua